src/main/java/Duke.java

```diff
@@ -1,10 +1,336 @@
+
+    static final String dukeKeywords = "\t\t Use 'todo', 'deadline' , 'event' to enter tasks!\n" +
+            "\t\t Use 'list' to view tasks!\n";
+    public static final String LOGO = " ____        _        \n"
+            + "|  _ \\ _   _| | _____ \n"
+            + "| | | | | | | |/ / _ \\\n"
+            + "| |_| | |_| |   <  __/\n"
+            + "|____/ \\__,_|_|\\_\\___|\n";
+
+    static final String dukeGreeting =
+            String.format("%s%s\t Hello! I'm Duke - your personal task manager \n%s%s",
+                    LOGO,
+                    lineDivider,
+                    dukeKeywords,
+                    lineDivider);
+    static final String dukeFarewell = String.format("%s\t Bye. Hope to see you again soon!\n%s",
+            lineDivider,
+            lineDivider);
+
+    static Scanner sc = new Scanner(System.in);
+
+    public static ArrayList<Task> taskList = new ArrayList<>();
+
+    public static String extractTaskMessage(String userInput) throws DukeExceptionNoTaskMessage {
+        if (userInput.indexOf(" ") > 0) {
+            return userInput.substring(userInput.indexOf(" ") + 1);
+        } else {
+            String errMessage = String.format("%s\tOOPS!!! The description of a task cannot be empty.\n%s",
+                    lineDivider,
+                    lineDivider);
+            throw new DukeExceptionNoTaskMessage(errMessage);
+        }
     }
+
+    public static String extractDateBy(String content) throws DukeExceptionNoDateBy {
+        if (content.contains("//")) {
+
+            // using the limit parameter limits the number outputs the string splits
+            String taskAndDate = content.split("//", 2)[1].strip();
+
+            if (taskAndDate.contains("by") || taskAndDate.contains("at")) {
+                return taskAndDate.substring(taskAndDate.indexOf(" ")).strip();
+            } else {
+                return taskAndDate;
+            }
+        } else {
+            String errMessage = String.format("%s\t OOPS!!! For deadlines and events, " +
+                            "there must be a dateby denoted using\n" +
+                            "\t  the '//' symbol (e.g \"deadline task 1 // by Tuesday 2pm\"\n " +
+                            "\t  or \"event task 1 // at 10/2/2021\" ) \n %s",
+                    lineDivider,
+                    lineDivider);
+            throw new DukeExceptionNoDateBy(errMessage);
+
+        }
+    }
+
+    public static void taskApp() throws DukeExceptionNoTaskMessage, DukeExceptionNoDateBy {
+
+//        ArrayList<Task> taskList = new ArrayList<>();
+        while (true) {
+
+            System.out.print("Enter command: ");
+            String userInput = sc.nextLine();
+
+            String commandWord = userInput.split(" ")[0].toLowerCase();
+            String content = null;
+
+            switch (commandWord) {
+                case "todo":
```

Comment:
I think indentation of case statements is wrong. Could refer to coding standards